src/index.spec.ts

```diff
@@ -236,9 +302,9 @@ describe('EppoClient E2E test', () => {
     };
 
     it('retries initial configuration request before resolving', async () => {
-      td.replace(HttpClient.prototype, 'get');
+      td.replace(HttpClient.prototype, 'getUniversalFlagConfiguration');
```

Comment:
our http client interface now has two methods, one for the UFC and one for bandit model parameters